graylog2-server/src/main/java/org/graylog2/rest/models/streams/requests/UpdateStreamRequest.java

```diff
@@ -42,15 +42,16 @@
     @Nullable
     public abstract String matchingType();
 
+    @JsonProperty("remove_from_all_messages")
```

Comment:
I personally find `remove_from_all_messages` pretty confusing. How about using `remove_matches_from_default_stream`? To me this makes more sense.

In the UI I would use "remove matches from all messages stream" or something like that, though.